src/Command/MigrateCommand.php

```diff
@@ -52,13 +58,20 @@ public function execute(InputInterface $input, OutputInterface $output)
             exit(1);
         }
 
+        $output->writeln("NOTE: Running module migrations is currently an experimental feature.");
+        $output->writeln("      You can skip this by using '--skip-module-migration'.");
         $output->writeln('Running migration scripts');
 
         $debugOutput = $input->getOption('verbose')
             ? $output
             : new NullOutput();
 
         /** @var MigrationHandler $oMigrationHandler */
```

Comment:
Please move this docblock above line 75 where we declare the `$oMigrationHandler`

Reply:
Done.

Reply:
Sorry, I was not clear enough. I was talking about the docblock
```
/** @var MigrationHandler $oMigrationHandler */
```
It should be directly above the line where we introduce the variable `$oMigrationHandler`.

The code should look like this:
```php
/** @var MigrationHandler $oMigrationHandler */
$oMigrationHandler = Registry::get(MigrationHandler::class);
$oMigrationHandler->run($timestamp, $debugOutput);
```

Reply:
Sorry for confusion. I have corrected it.  Hope now codes are good.